### ISSUE TITLE
Exports?

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/HistoryLog#11]
Is it possible to display exports in the Item History Log? Specifically thinking it would be nice to have any exports using MerrittLink display in the log.

**Comment by NedHenry** [comment]
This makes sense for exports using MerrittLink, which submit to an external
archiving service. I can implement that without too much difficulty.
I do not think this makes as much sense for exports using MetsExport,
because the METS files associated with items or collections may get public
urlsand so the admin does not control when a METS file is generated
(exported). I think the same argument applies to XML and RSS exports.
Let me know if you can think of any other types of exports that it would
make sense to record in administrative logs.
Best,
Ned

**Comment by JessWaggs** [comment]
I agree. I think MerrittLink exports are the only type I can think of at this point that we use.
